### PR TITLE
GRD-85034

### PR DESCRIPTION
**Pull request by apurva-birajdar**
Publish all internal changes to public. 
Verified by QA .

### CODE REVIEW

**Review by chirag-ibm — APPROVED** [review]

File: filter-plugin/logstash-filter-aurora-mysql-guardium/auroraMysqlCloudwatch.conf [hunk 1]
```diff
@@ -17,31 +17,36 @@ input{
 		event_filter => ""
 		type => "AuroraMysql"
 		add_field => {"account_id" => "<account_id>"}
+        #Unmask is an optional parameter. To get unmasked logs, you need to set unmask = true.
```
Comment (PratikshaSonawane) [inline filter-plugin/logstash-filter-aurora-mysql-guardium/auroraMysqlCloudwatch.conf:20]: Please remove these lines this fix is yet to be tested

**Review by PratikshaSonawane — CHANGES_REQUESTED** [review]

**Comment by ofer-haim** [comment]
Shouldn't we upgrade the plugin version?

File: filter-plugin/logstash-filter-aurora-mysql-guardium/auroraMysqlCloudwatch.conf (hunk 1 above)
Comment (apurva-birajdar) [reply filter-plugin/logstash-filter-aurora-mysql-guardium/auroraMysqlCloudwatch.conf]: done.

**Comment by apurva-birajdar** [comment]
> Shouldn't we upgrade the plugin version?

Hi Ofer, We have not updated zip its just few checks that we added and removed the incorrect ones.

**Review by PratikshaSonawane — APPROVED** [review]

**Pull request merged**